src/applications/vaos/new-appointment/redux/reducer.js

```diff
@@ -18,6 +18,8 @@ import {
   FORM_UPDATE_FACILITY_TYPE,
   FORM_PAGE_FACILITY_OPEN_SUCCEEDED,
   FORM_PAGE_FACILITY_OPEN_FAILED,
+  FORM_PAGE_FACILITY_V2_OPEN,
+  FORM_PAGE_FACILITY_V2_OPEN_SUCCEEDED,
```

Comment:
I think we also need a reducer for the V2 failed action

Reply:
Reducer will be added as part of eligibility check PR